### DISCUSSION TITLE
Beginner Q: Next step for lnrrpc

### DISCUSSION THREAD

**Discussion by urPlaceOrMine** [lightningnetwork/lnd#6700] [Q&A] [answered]
LND version: v0.14.3-beta
my end goal it to connect Bolztd to my LND service. When I try to connect to LND the LND console is reporting this error:  "[ERR] RPCS: [/lnrpc.Lightning/GetInfo]: permission denied" 

These ports are registered in LND
2022-07-03 08:14:22.598 [INF] RPCS: RPC server listening on 192.168.1.250:10010
2022-07-03 08:14:22.598 [INF] RPCS: RPC server listening on 0.0.0.0:10009
2022-07-03 08:14:22.599 [INF] RPCS: gRPC proxy started at 0.0.0.0:8080

I did install Lnrpc using the lnrpc install instructions from here: https://github.com/lightningnetwork/lnd/tree/master/lnrpc
and I was able to complete the docker build. however I am not sure how to run LNRPC after the build. If I need to be running LNRPC  how do I start it.  I am not clear on the next step from the instructions.  

thinking the error above was access denied to a path I looked for a folder at lnrpc.lightening  with my user & root account.  If ti is supposed to be a path it does not exist. 

my LND.Conf is at the end of this message 

I think I am missing a fundamental step.  I am just not sure what it is. 

Thank you for your assistance.  

ur

LND.CONF file looks like this:
The TLS certs have been regenerated after making network changes. 

```
listen=0.0.0.0:9735 

# Tells LND to listen on all of your computer's interfaces
# This could alternatively be set to your router's subnet IP
# tlsextraip=0.0.0.0
tlsextraip=192.168.1.250
tlsextraip=127.0.0.1
tlsextraip=192.168.1.1
tlsextraip=1.2.3.4

# Tells LND where to listen for RPC messages
# This could also be set to your router's subnet IP

 rpclisten=0.0.0.0:10009 
rpclisten=192.168.1.250:10010
# rpclisten=192.168.1.250:10009
# rpclisten=127.0.0.1:10009

restlisten=0.0.0.0:8080
externalip=1.2.3.4
# 

# debuglevel=debug,NTFR=debug,IRPC=debug,ARPC=debug,SGNR=debug,WLKT=debug
maxlogfiles=3
maxlogfilesize=10 
```

**Comment by positiveblue** [comment, marked as answer]
@urPlaceOrMine I edited your message to format the `lnd.conf` data properly.

I never used Bolzt but did you load the macaroon/tlscerts so it can connect (and authenticate) with your LND?  

You can add the correct paths in the boltz config. [Check the last config fields here](https://docs.boltz.exchange/en/latest/deployment/)

**Reply by urPlaceOrMine** [reply]
Thank you for fixing the font on the post.  
& Thank you for the idea.  I am still working this item.  I will follow-up once I sort it out.

**Comment by urPlaceOrMine** [comment]
This is sorted out.  
Two items.  The error "[ERR] RPCS: [/lnrpc.Lightning/GetInfo]: permission denied" means the admin.macaroon can not be read; or in my case the wrong file was listed.  

The admin.macaroon file needs to have read permissions. 
`ls -l ~/path/admin.macaroon`  
will give you the permissions the file has. 
In my case the boltz.toml file has a line to point to the admin.macaroon.  
`macaroon = /path/admin.macaroon` 
However, in my case the line was pointing to the invoice.macaroon instead.  Once this was corrected it and put the right file there I was able to move past the error. 

Thank you for your assistance.
